baybe/searchspace/discrete.py

```diff
@@ -717,17 +732,20 @@ def _apply_polars_constraint_filter(
         constraints: Collection of discrete constraints.
 
     Returns:
-        A Polars lazyframe with undesired rows removed.
-
+        A tuple containing the Polars lazyframe with undesired rows removed and a
+        boolean mask indicating which constraints have NOT been applied.
```

Comment:
```suggestion
        A tuple containing

        * the Polars lazyframe with undesired rows removed
        * a Boolean mask indicating which constraints have **not** been applied.
```

Reply:
amended last commit

Reply:
Thx, but I saw that you removed the blank line. Needed, unfortunately, since otherwise stuff will not be rendered properly in the docs (I know, currently private, but I think we should still write the docstring in a way that is rendered properly)

Reply:
And: the `:` behind `A tuple containing` must go, since otherwise the stuff before is treated as `return type`

Reply:
saw it, didnt know about `:`
this is how it looks now (I also prefer an indent added)
![image](https://github.com/user-attachments/assets/7e922628-bf47-4022-9ea0-2ca07bf36efc)

Reply:
Just a question, since I saw that you changed `Boolean` back to `boolean`: I actually think that `Boolean` is correct (and we messed it up in other places), since the guy was called `Boole`. Similar like you also write `Gaussian` instead of `gaussian`

Reply:
<img width="726" alt="image" src="https://github.com/user-attachments/assets/cd0539fd-4f8c-4926-93f1-52dedcce57f9">

Reply:
<img width="969" alt="image" src="https://github.com/user-attachments/assets/9dcd0fa9-d29b-413c-a333-f867134ecdd3">

Reply:
ok amended